examples/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ target_link_libraries(mlmc smoothg ${TPL_LIBRARIES})
 
 # add tests
 if (NOT DEFINED SMOOTHG_TEST_PROCS)
-    set(SMOOTHG_TEST_PROCS 4)
+    set(SMOOTHG_TEST_PROCS 2)
```

Comment:
@barker29 The test results on parallel graph now depend on number of processors, so I match this with the value set on travis.